src/Selenium2Library/keywords/_element.py

```diff
@@ -267,13 +267,15 @@ def element_text_should_be(self, locator, expected, message=''):
                           "in fact it was '%s'." % (locator, expected, actual)
             raise AssertionError(message)
 
-    def get_element_attribute(self, attribute_locator):
+    def get_element_attribute(self, locator, attribute_name=None):
         """Return value of element attribute.
```

Comment:
I've updated documentation according to your requirements. If you will not satisfied it again, please, help me and propose examples how to fix it.